pkg/constants/http.go

```diff
@@ -36,7 +36,8 @@ const (
 	LedgerEndpoint = "/ledger/"
 	GetBlockHeader = "/ledger/block/{blockId:[0-9]+}"
 	GetPath        = "/ledger/path"
-	GetTxProof     = "/ledger/proof/{blockId:[0-9]+}"
+	GetTxProof     = "/ledger/proof/tx/{blockId:[0-9]+}"
+	GetDataProof   = "/ledger/proof/data/{blockId:[0-9]+}/{dbname:" + `[0-9a-zA-Z_\-\.]+` + "}/{key}"
```

Comment:
I think that if the user make a request to `/ledger/proof/tx` `/ledger/proof/tx/` `/ledger/proof/data` or `/ledger/proof/data/` he will get an unformatted error response content from within `gorilla/mux` which is not interpreted very well by the SDK.

Reply:
It will get default 404 response. I can add multiple invalid handlers to take care of following cases: 
1. ` /ledger/proof/tx`
2. `/ledger/proof/data`
3. `/ledger/proof/data/{blockId}`
4. `ledger/proof/data/{blockId}/{dbname}`
But once user uses sdk, no such thing is possible. wdut?

Reply:
I think it makes sense to cover all the corners, after all, the REST spec is public and user/community can write her own SDK.... especially in other languages, like Java, Node, who knows...